core/src/main/scala/org/apache/spark/deploy/worker/Worker.scala

```diff
@@ -362,11 +362,13 @@ private[spark] class Worker(
             self,
             workerId,
             host,
+            webUiPort,
```

Comment:
Oh no. I thought I addressed all the comments including this one. Do you want me to fix it?